### ISSUE TITLE
Hi how to see fc layer's weights?

### ISSUE THREAD

**Issue by markofan** [ehosseiniasl/3d-convolutional-network#8]
After I finetuned my scae, I got my scae_AD_Normal.pkl, but I find that only 2 elements in there(I only use one layer CAE1+2 layers fc+softmax), the size of it  are (8,3,1,3,3) and (8,), when I use load_fc() function, it goes wrong, when did the fc weights save?

**Comment by markofan** [comment]
When I use scae.load(scae_model) to load my finetuned model, but when I use the training-data to test, still can't the result, the model are classified to one class. Is there something wrong with load? I am confused!
